Lesson_19/custom-map/method-map.js

```diff
@@ -0,0 +1,35 @@
+document.addEventListener('DOMContentLoaded', () => {
+   // VARIANT 1 - accepts only a callback function and is available in the array prototype
+   function myMap(cb) {
+      const newArr = [];
+      
+      for(let i = 0; i < this.length; i++) {
+         newArr.push( cb(this[i], i, this) );
```

Comment:
good, but its better for code style to create variables with calculated value `cb(this[i], i, this)`
```javascript
for(let i = 0; i < this.length; i++) {
  const newElement = cb(this[i], i, this);
  newArr.push(newElement);
}
```